config/crd/bases/kahu.io_restores.yaml

```diff
@@ -0,0 +1,195 @@
+---
```

Comment:
we can keep config/crd/v1beta1/bases as the folder

Reply:
updated as per comment